src/organizations/components/OrgProfileTab/index.tsx

```diff
@@ -16,42 +16,42 @@ import CopyableLabeledData from 'src/organizations/components/OrgProfileTab/Copy
 import DeletePanel from 'src/organizations/components/OrgProfileTab/DeletePanel'
 
 // Utils
-import {getOrg} from 'src/organizations/selectors'
 import {isFlagEnabled} from 'src/shared/utils/featureFlag'
 import {CLOUD} from 'src/shared/constants'
+import {shouldUseQuartzIdentity} from 'src/identity/utils/shouldUseQuartzIdentity'
+import 'src/organizations/components/OrgProfileTab/style.scss'
```

Comment:
To keep this clear, let's consider separating these imports out with different comments:

`// Utils` for for the feature flag in utils; `// Constants` for the constants, `// Stylesheets` or equivalent for `scss.